data/pi_data_formatter.py

```
@@ -0,0 +1,550 @@
+"""
+PI Data Formatter
+
+Converts nuclear plant simulation data to industry-standard PI (Plant Information) format
+used in real power plants. This module provides PI tag naming conventions, data quality
+indicators, and export formats that match what operators see in control rooms.
+"""
+
+import pandas as pd
+import numpy as np
+from datetime import datetime, timezone
```

Ruff (0.11.9)

`datetime.timezone` imported but unused

Remove unused import: `datetime.timezone`

(F401)